app/controllers/items_controller.rb

```diff
@@ -78,14 +78,22 @@ def restore
 
   private
 
-  def clean_purchase_amount
+  def clean_item_value_in_cents
```

Comment:
Is there anywhere we're still entering value in cents? I'd love if we could kill this function in it's entirety - users should only ever be aware of dollars, anyways.

Reply:
It looks like only one other place that is left, which is `Donation`'s `money_raised`. There is a PR for that already. It can be found by looking at the mentions/links of this issue's page. However, it is handled very differently (by migrating the DB).

Should we continue on handling it in this PR also?

Reply:
If you could change it there as well that would be great - we really don't want to migrate the DB for these.